docs-src/python/understanding-deprecated-patches-in-python-sdk.md

```diff
@@ -0,0 +1,35 @@
+---
+id: understanding-deprecated-patches-in-python-sdk
+title: Understanding deprecated Patches in the Python SDK
+sidebar_label: Deprecated Patches
+description: An overview of deprecated patches in the Python SDK, detailing their role in ensuring smooth transitions between workflow versions and preventing replay failures.
+tags:
+- deprecated patches
+- python sdk
+- workflow transition
+- best practices
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/edu-1171/version_your_workflows/workflow_3_patch_deprecated_dacx.py. -->
+
+After ensuring that all Workflows started with `pre_patch_activity` code have finished, you can [deprecate the patch](https://python.temporal.io/temporalio.workflow.html#deprecate_patch).
+
+Deprecated patches serve as a bridge between `pre_patch_activity` and `post_patch_activity`. They function similarly to regular patches by adding a marker to the Workflow history. However, this marker won't cause a replay failure when the Workflow code doesn't produce it.
+
+If, during the deployment of `post_patch_activity`, there are still live Workers running `pre_patch_activity` code and these Workers pick up Workflow histories generated by `post_patch_activity`, they will safely use the patched branch.
```

Comment:
May be out of scope for this so feel free to ignore, but my first reaction to reading this was "Oh cool! Why?" so if we can expand on how this makes pre_patch_activity's safe to use with post_patch_activity code that would be great. But if that's a huge explanation or way out of scope for this guide I'd say lets make a note to cover it elsewhere and move on.